### ISSUE TITLE
listing find: examples from online help don't work

### ISSUE THREAD

**Issue by unsystemizer** [17Q4MX2hmktmpuUKHFuoRmS5MfB5XPbhod/dropzone_ruby#21]
```
$ dropzone --force_testnet listing find 2880 --trace
/usr/local/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/abstract_response.rb:76:in `return!': HTTP status code 522 (RestClient::RequestFailed)
        from /usr/local/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:495:in `process_result'
        from /usr/local/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:421:in `block in transmit'
        from /usr/local/lib/ruby/2.2.0/net/http.rb:853:in `start'
        from /usr/local/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'
        from /usr/local/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'
        from /usr/local/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'
        from /usr/local/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/resource.rb:51:in `get'
        from /usr/local/lib/ruby/gems/2.2.0/gems/dropzone_ruby-0.1.3/lib/blockrio_ext.rb:32:in `getblock'
        from /usr/local/lib/ruby/gems/2.2.0/gems/dropzone_ruby-0.1.3/lib/dropzone/connection.rb:70:in `messages_in_block'
        from /usr/local/lib/ruby/gems/2.2.0/gems/dropzone_ruby-0.1.3/lib/dropzone/item.rb:81:in `block in find_creates_since_block'
        from /usr/local/lib/ruby/gems/2.2.0/gems/dropzone_ruby-0.1.3/lib/dropzone/item.rb:80:in `downto'
        from /usr/local/lib/ruby/gems/2.2.0/gems/dropzone_ruby-0.1.3/lib/dropzone/item.rb:80:in `each'
        from /usr/local/lib/ruby/gems/2.2.0/gems/dropzone_ruby-0.1.3/lib/dropzone/item.rb:80:in `collect'
        from /usr/local/lib/ruby/gems/2.2.0/gems/dropzone_ruby-0.1.3/lib/dropzone/item.rb:80:in `find_creates_since_block'
        from /usr/local/lib/ruby/gems/2.2.0/gems/dropzone_ruby-0.1.3/lib/dropzone/command.rb:370:in `listing_find'
        from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:179:in `call'
        from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
        from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
        from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/runner.rb:68:in `run!'
        from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
        from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/import.rb:5:in `block in <top (required)>'
```

**Comment by unsystemizer** [comment]
```
$ bitcoin-cli -testnet getinfo
{
    "version" : 110200,
    "protocolversion" : 70002,
    "walletversion" : 60000,
    "balance" : 0.03412850,
    "blocks" : 629879,
    "timeoffset" : 0,
    "connections" : 4,
    "proxy" : "",
    "difficulty" : 1.00000000,
    "testnet" : true,
    "keypoololdest" : 1440945664,
    "keypoolsize" : 101,
    "paytxfee" : 0.00000000,
    "relayfee" : 0.00005000,
    "errors" : "WARNING: check your network connection, 0 blocks received in the last 4 hours (24 expected)"
}

$ dropzone --force_testnet listing find 1 --start_at 629875 --debug
RestClient.get "http://tbtc.blockr.io/api/v1/block/info/629875", "Accept"=>"*/*; q=0.5, application/xml", "Accept-Encoding"=>"gzip, deflate", "Content-Type"=>"application/json"
11:28:18 Connecting directly to tbtc.blockr.io:80
11:28:19 Connected to tbtc.blockr.io:80
# => 200 OK | application/json 621 bytes
RestClient.get "https://blockchain.info/block-index/000000006276cbf80c16535f12d9c44d37b076bbd7ae259815e70da7ca8f4ecf?format=json", "Accept"=>"*/*; q=0.5, application/xml", "Accept-Encoding"=>"gzip, deflate", "Content-Type"=>"application/json"
11:28:19 Connecting directly to blockchain.info:443
11:28:19 Connected to blockchain.info:443
# => 500 InternalServerError | text/plain 15 bytes
error: 500 Internal Server Error. Use --trace to view backtrace

```

**Comment by ScroogeMcDuckButWithBitcoin** [comment]
This is because of the --force_testnet option. 

Blockchain.info is being used as the connection 'driver' for all listing finds, and blockchain.info itself doesn't support testnet. (Which is weird, because the driver name is 'BlockrIo'...)

I don't think I want to spend time on implementing a driver architecture the way I did for the javascript version, so, I changed this to use sochain for this call instead when on testnet. Fixed.
